radar.csv

```diff
@@ -108,8 +108,16 @@ Physical whiteboards,adopt,tools,true,"Some teams use physical whiteboards for i
+"Kanban",explore,techniques,true,"Flow-based agile framework the emphasises limiting work-in-progress (WIP). Historically adopted by teams who find time-boxed iterations restrictive, or need to work in a more reactive manner. Currently being explored by ICloneable&lt;Sheep&gt;."
 "MoSCoW",retire,techniques,true,"MoSCoW stands for Must, Should, Could and Wont. This prioritisation technique requires the majority of features / stories to be identified up front - this does not fit well with our exploratory approach to finding customer needs and drilling in based on our riskiest assumption. It also has the effect of creating fixed view of what will be delivered and can result in teams delivering work that is not valued by customers."
 "master is always releasable",adopt,techniques,true,"'master as always releasable' is an ethos teams will follow and constantly strive to make true. We should have enough confidence in the product and automated tests that anyone can press the release button at any time. We release whenever we have something to release. We should act like master is always releasable at any time, and when we discover that it isn't/wasn't releasable, reflect on why and add the appropriate checks for next time."
+"Zero bugs policy",explore,techniques,true,"A <a href='https://medium.com/qualityfaster/the-zero-bug-policy-b0bd987be684'>zero-bug policy</a> does not mean bug-free code production; it means striving to eradicate all known bugs. ICloneable&lt;Sheep&gt; will be exploring this with support from the Quality Coaches."
+"Karma",adopt,languages and frameworks,true,"<a href='https://karma-runner.github.io/latest/index.html'>Karma</a> is a test runner for JavaScript which spawns a web server that executes source code against test code for each of the browsers connected."
+"jasmine",explore,languages and frameworks,true,"<a href='https://jasmine.github.io/'>Jasmine</a> is a BDD framework for testing JavaScript code. The Monitor team are using it through Karma."
+"nightwatch",explore,languages and frameworks,true,"<a href='http://nightwatchjs.org/'>Nightwatch</a> is an end-to-end testing solution for browser-based apps using WebDriver. The ICloneable&lt;Sheep&gt; team is exploring this."
+"jest",adopt,languages and frameworks,true,"<a href='https://jestjs.io/'>Jest</a> is a way to test all JavaScript code that's used by Facebook. COREDev have a non-trivial amount of tests over multiple projects that use jest."
+"spectron",explore,languages and frameworks,true,"<a href='https://github.com/electron/spectron'>spectron</a> is a tool to allow you to easily test your Electron apps using ChromeDriver/WebDrive IO. Figleaf is exploring this."
+"mocha",explore,languages and frameworks,true,"<a href='https://mochajs.org'>mocha</a> is a feature rich test framework running on Node/Browser. The ICloneable&lt;Sheep&gt; team are exploring this."
```

Comment:
In IClonableSheep we are currently using `mocha` and `chai`, we are exploring `jest`.

Reply:
Hmm. Should I put `mocha` and `chai` in adopt? (as in your confident that it's a great default choice) or should I leave them in `explore` as it's possible you might choose `jest` to be in adopt?

Reply:
I agree. From my experience `jest` is a good all purpose testing framework, so it doesn't make sense to keep `mocha `and `chai` in addition to it if we don't have any strong reasons for that.